pkgs/tools/system/acpica-tools/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, bison, flex }:
+
+stdenv.mkDerivation rec {
+  name = "acpica-tools-${version}";
+  version = "20180209";
+
+  src = fetchurl {
+    url = "https://acpica.org/sites/acpica/files/acpica-unix-${version}.tar.gz";
+    sha256 = "1rpdfwa4vwnvyxdp9ygqjckmabc3s8kyg3jyq4n4f0rhr1zl4zy5";
+  };
+
+  NIX_CFLAGS_COMPILE = "-O3";
+
+  enableParallelBuilding = true;
+
+  buildFlags = stdenv.lib.concatStringsSep " " [
+    "acpibin"
+    "acpidump"
+    "acpiexec"
+    "acpihelp"
+    "acpinames"
+    "acpixtract"
+  ];
+
+  buildInputs = [ bison flex ];
```

Comment:
Both of these can be treated as `nativeBuildInputs`, if they are used only for codegen during the build.

Reply:
Thanks, fixed.